app/assets/javascripts/external_pledges.coffee

```diff
@@ -1,9 +0,0 @@
-# Place all the behaviors and hooks related to the matching controller here.
-# All this logic will automatically be available in application.js.
-# You can use CoffeeScript in this file: http://coffeescript.org/
-
-jQuery ->
-  # use event that works with rails turbolinks
-  $(document).on 'turbolinks:load', ->
-    # select all elements with id ending with "modal"
-    $('[id$=modal]').modalSteps()
```

Comment:
This is not actually used, near as I can tell? So we can save ourselves the hassle and just remove it.